pkgs/by-name/bi/bitbox-bridge/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  pkgs,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "bitbox-bridge";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "BitBoxSwiss";
+    repo = "bitbox-bridge";
+    rev = "v${version}";
+    fetchSubmodules = true;
+    hash = "sha256-+pMXWXGHyyBx3N0kiro9NS0mPmSQzzBmp+pkoBLH7z0=";
+  };
+
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-6vD0XjGH1PXjiRjgnHWSZSixXOc2Yecui8U5FAGefBU=";
+
+  nativeBuildInputs = [
+    pkgs.pkg-config
+  ];
+
+  buildInputs = [
+    pkgs.libudev-zero
+  ];
+
+  doCheck = false;
+
+  meta = with stdenv.lib; {
```

Comment:
please do not use with lib; or anything like that. It increases eval times and is generally bad practice

Reply:
I've removed it, however just for my curiousity & understanding, could you elaborate why it increases eval times? I'd appreciate a reference to the good practice(s) as well, it is my first `nixpkgs` contribution! :)